core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -79,6 +80,11 @@ object KafkaRaftManager {
     Files.createDirectories(dir.toPath)
     dir
   }
+
+  private def deleteLogDirectory(logDir: File, logDirName: String): Unit = {
+    val dir = new Directory(new File(logDir.getAbsolutePath, logDirName))
+    dir.deleteRecursively()
+  }
```

Comment:
This code should not be in `core/src/main` since it is not used by `core/src/main`.